tests/lax_numpy_test.py

```diff
@@ -1473,6 +1473,20 @@ def testDiagFlat(self, shape, dtype, k):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}".format(
+          jtu.format_shape_dtype_string(shape, dtype)),
+       "dtype": dtype, "shape": shape}
+      for dtype in default_dtypes
+      for shape in one_dim_array_shapes))
+  def testPolyMul(self, shape, dtype):
+    rng = jtu.rand_default(self.rng())
+    np_fun = lambda arg1, arg2: np.polymul(arg1, arg2)
+    jnp_fun = lambda arg1, arg2: jnp.polymul(arg1, arg2)
+    args_maker = lambda: [rng(shape, dtype), rng(shape, dtype)]
+    self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
```

Comment:
Use a version of jnp_fun with `trim_leading_zeros=True` here so that the test will (presumably) pass.

(below, `trim_leading_zeros` will have to remain `False`, or else the compilation test will fail)